pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -484,7 +485,12 @@ private static void setupTaskReceiver(Config cfg, String projectId) {
         if (enableHKeepTasks) {
             var subName = ProjectSubscriptionName.of(projectId, cfg.getString(ConfigFile.PUBSUB_HKEEP_TASKS_SUB));
             var receiver = new HousekeepingTaskReceiver(subName, scheduler);
-            taskSubscriber = Subscriber.newBuilder(subName, receiver).build();
+            taskSubscriber = Subscriber.newBuilder(subName, receiver)
+                    .setParallelPullCount(1)
+                    .setExecutorProvider(InstantiatingExecutorProvider.newBuilder()
+                            .setExecutorThreadCount(1)
+                            .build())
```

Comment:
Guess default for this one is 6?

Reply:
Looks like the default is variable: `5 * num_processors`. So at least 5.